### PR TITLE
[Algorithm] Update DDPG Example

### PR DESCRIPTION
**Pull request by BY571**
## Description

Similar to #1523 

## Motivation and Context

Why is this change required? What problem does it solve?
If it fixes an open issue, please link to the issue here.
You can use the syntax `close #15213` if this solves the issue #15213

- [ ] I have raised an issue to propose this change ([required](https://github.com/pytorch/rl/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Remove all that do not apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)
- [ ] Example (update in the folder of examples)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] I have read the [CONTRIBUTION](https://github.com/pytorch/rl/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: examples/ddpg/config.yaml [hunk 1]
```diff
@@ -1,45 +1,47 @@
-# Environment
+# environment and task
 env:
   name: HalfCheetah-v3
   task: ""
-  exp_name: "HalfCheetah-DDPG"
+  exp_name: "HalfCheetah-DDPG-ICLR"
```
Comment (vmoens) [inline examples/ddpg/config.yaml:5]: ditto

File: examples/ddpg/ddpg.py [hunk 2]
```diff
@@ -43,137 +46,150 @@ def main(cfg: "DictConfig"):  # noqa: F821
...
+            tensordict["next", "done"]
+        ]
+
+        # Logging
+        if len(episode_rewards) > 0:
+            episode_length = tensordict["next", "step_count"][
+                tensordict["next", "done"]
+            ]
+            logger.log_scalar(
+                "train/reward", episode_rewards.mean().item(), collected_frames
```
Comment (vmoens) [inline examples/ddpg/ddpg.py:157]: ditto, let's put the logging in a separate function

**Review by vmoens — APPROVED** [review]
LGTM 
same comments as others

File: examples/ddpg/ddpg.py (hunk 2 above)
Comment (BY571) [reply examples/ddpg/ddpg.py:157]: What do you think of the updated version for logging?

**Review by vmoens — APPROVED** [review]
Awesome I love it! Much clearer

**Review by vmoens — COMMENTED** [review]
The Examples CI is broken, do you think you can fix it?

**Review by vmoens — APPROVED** [review]
LGTM thanks for this!

**Pull request merged**